extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -229,8 +230,15 @@ export class JupyterServerInstallation implements IJupyterServerInstallation {
 								return reject(err);
 							}
 						}
-						decompress(pythonPackagePathLocal, installPath).then(files => {
-							//Delete zip/tar file
+						if (utils.getOSPlatform() === utils.Platform.Windows) {
```

Comment:
Instead of having this extra logic what if we just put the windows files in a tar.gz too? We're the ones hosting this content right?

Reply:
We use zip on Windows because we leverage these same compressed files for offline installation scenarios. Users on older version of Windows can't open tar files by default.

Second, I'm worried about changing this during Ask mode. If we don't need to change the compression format, let's not do that right now.